README.md

```diff
@@ -22,19 +22,32 @@ Below are some common usage examples:
 
 ```shell
 # Build images of the hello-world containers
-bake -p hello-world -i 0.0.0-dev
+bake --product hello-world
 
 # Build only one version [0.37.2] of OPA
-bake -p opa=0.37.2 -i 0.0.0-dev
+bake --product opa=0.37.2
+
+# Dry run. Do not buikd anything. Print the the generated Bakefile.
+bake --product hello-world --dry
+
+# Build all OPA images and set the organisation to "sandbox"
```

Comment:
This comment is wrong, it only builds 0.37.2.
I guess either the comment or the command should be updated.

Reply:
Fixed https://github.com/stackabletech/image-tools/pull/38/commits/eb5f658faf1610d743bdb265726cb6cba569e594